spec/intl.html

```diff
@@ -1340,6 +1340,270 @@ <h1>CalendarEraYear ( _calendar_, _dateLike_ )</h1>
             1. Return _result_.
           </emu-alg>
         </emu-clause>
+
+        <emu-clause id="sec-temporal-calendardatetoiso" type="abstract operation">
+          <h1>
+            CalendarDateToISO (
+              _calendar_: a String,
+              _fields_: a Record,
```

Comment:
This isn't a Record - it's an Object, returned from PrepareTemporalFields. You could do as in #2089 and say something like "an ordinary Object with no more and no less than the own data properties listed in [Table 13](https://tc39.es/proposal-temporal/#table-temporal-field-requirements)"

Reply:
Done.